### PR TITLE
Improve Active-Active docker

### PR DESCRIPTION
**Pull request by bmansheim**

### CODE REVIEW

File: content/embeds/docker-memory-limitation.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+If you cannot activate the database because of a memory limitation,
+make sure that Docker has enough memory allocated in the Advanced section
+of Docker Settings.
```
Comment (banker) [inline content/embeds/docker-memory-limitation.md:3]: Please use the appropriate convention to call out "Advanced" and "Docket Settings". This is a place where it'd be useful to link to the official Docker documentation.

File: content/rs/getting-started/getting-started-active-active.md [hunk 2]
```diff
@@ -92,10 +92,10 @@ Now we have two Redis Enterprise Software clusters with FQDNs
     1. In the **participating clusters** list, add the address and admin credentials for:
         - `https://cluster1.local:9443` - the cluster you are currently connected to
         - `https://cluster2.local:9443` - the other cluster
-    <!-- Also in create-crdb.md -->
+
     1. In the **Database clustering** option, either:
 
-        - Make sure the Database clustering is enabled and select the number of shards
+        - Make sure that Database clustering is enabled and select the number of shards
```
Comment (banker) [inline content/rs/getting-started/getting-started-active-active.md:98]: "Database clustering" needs to be in bold I think?

File: content/rs/getting-started/getting-started-active-active.md [hunk 3]
```diff
@@ -92,10 +92,10 @@ Now we have two Redis Enterprise Software clusters with FQDNs
     1. In the **participating clusters** list, add the address and admin credentials for:
         - `https://cluster1.local:9443` - the cluster you are currently connected to
         - `https://cluster2.local:9443` - the other cluster
-    <!-- Also in create-crdb.md -->
+
     1. In the **Database clustering** option, either:
```
Comment (banker) [inline content/rs/getting-started/getting-started-active-active.md:96]: "In" strikes me as the wrong preposition here. We usually don't "In the X option". Maybe "For"?

File: content/rs/getting-started/getting-started-active-active.md [hunk 4]
```diff
@@ -109,13 +109,12 @@ Now we have two Redis Enterprise Software clusters with FQDNs
...
-    Note: If you cannot activate the database because of a memory limitation,
-    make sure that Docker has enough memory allocated in the Advanced section
-    of Docker Settings.
+    {{< note >}}
+    {{< embed-md "docker-memory-limitation.md" >}}
+    {{< /note >}}
 
-1. After the Active-Active database is created, you can now visit each cluster 1 at
-https://localhost:8443 and cluster 2 at https://localhost:8445.
+1. After the Active-Active database is created, access the RS admin console
```
Comment (banker) [inline content/rs/getting-started/getting-started-active-active.md:116]: If there's no other admin console, why do we need to quality this with "RS"?

File: content/rs/getting-started/getting-started-active-active.md (hunk 3 above)
Comment (bmansheim) [reply content/rs/getting-started/getting-started-active-active.md:96]: This is a comment to indicate that the same content is used in another article. It doesn't show in the HTML.

File: content/embeds/docker-memory-limitation.md (hunk 1 above)
Comment (bmansheim) [reply content/embeds/docker-memory-limitation.md:3]: Unfortunately it's different per OS. Do you think we should have links for each OS?
Currently it's written for Windows which itself is a problem.
I would lean toward making it more generic and that customers should go consult the Docker docs if they need to. I guess we can link to https://docs.docker.com/.

File: content/rs/getting-started/getting-started-active-active.md (hunk 4 above)
Comment (bmansheim) [reply content/rs/getting-started/getting-started-active-active.md:116]: I would treat it as a product name, but I can hear the argument against also.
Should it be Admin Console or admin console?

**Pull request merged**